app.js

```diff
@@ -0,0 +1,10 @@
+const db = require("./db/connection");
+const express = require("express");
+const seed = require("./db/seeds/seed");
+const getCategories = require("./controllers/app-controllers");
+const app = express();
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
```

Comment:
Error handling middleware for 500 should be included.

A non existent path one also to be included at some point after Jim's lecture this morning, not needed for this ticket but just a heads up.